langlib/lang.regexp/src/main/ballerina/regexp.bal

```diff
@@ -53,6 +53,16 @@ type GroupsArrayType GroupsAsSpanArrayType[];
 
 # Returns the first match of a regular expression within a string.
 #
+# ```ballerina
+#var sampleRegex = re `World`;
```

Comment:
```suggestion
# var sampleRegex = re `World`;
```

Please fix everywhere.

Reply:
Can we use a different variable name? Maybe even `r` is OK given how we've written other examples.